### DISCUSSION TITLE
(Projects Beta) Include 'Back to Projects' Button

### DISCUSSION THREAD

**Discussion by amagalla1394** [community/community#9597] [Projects and Issues]
When working within the Projects Beta space, I noticed that there was no way for me to return to the 'Projects' page. I assume many others have also noticed.
Please add a button to make transition easier than the back arrow or the account dropdown option. 
Thank you.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
